libraries/Microsoft.Bot.Builder.AI.QnA/QnAMaker.cs

```diff
@@ -85,8 +86,15 @@ public QnAMaker(QnAMakerEndpoint endpoint, QnAMakerOptions options, HttpClient h
             TelemetryClient = telemetryClient ?? new NullBotTelemetryClient();
             LogPersonalInformation = logPersonalInformation;
 
-            this._generateAnswerHelper = new GenerateAnswerUtils(TelemetryClient, _endpoint, options, _httpClient);
-            this._activeLearningTrainHelper = new TrainUtils(_endpoint, _httpClient);
+            if (string.Equals(endpoint.QnAServiceType, Constants.LanguageQnaServiceType, StringComparison.OrdinalIgnoreCase))
```

Comment:
We may need to think about this a bit more, the class now encapsulates two different service types. It may be better to refactor the specific logic for each service type out of the main class into smaller internal classes so we can eventually deprecate the old service. 

We seem to be doing the `string.Equals(endpoint.QnAServiceType, Constants.LanguageQnaServiceType, StringComparison.OrdinalIgnoreCase)` in many places on this class and that will become harder to maintain down the road.